client/src/components/application/Examine/nwpta/nwpta.vue

```diff
@@ -1,6 +1,6 @@
 <!-- eslint-disable -->
 <template>
-  <fragment name="jurisdiction+'-nwpta-fragment'">
+  <div name="jurisdiction+'-nwpta-fragment'">
```

Comment:
It looks like the fragment element was used to have a single root element (a Vue requirement).

However, a div works just fine, so I'm wondering why this was done in the first place and if there's more to it than this ???

Ref: https://www.npmjs.com/package/vue-fragments

Reply:
Maybe it's just to avoid an extra `div` level in the component. Therefore, please verify that the output visually is OK (and possibly inspect the div to see if any padding/margin are added or anything else undesired).